src/middleware/visualizer.coffee

```diff
@@ -72,13 +114,13 @@ storeVisualizerEvents = (ctx, done) ->
 
   # Transaction end for primary route
   trxEvents.push new events.VisualizerEvent
-    ts: endTS
+    ts: endTS + orchestrationTSBufferMillis
```

Comment:
I don't think we should add the orchestration buffer when enableTSNormalization is false. What you could do is just set `orchestrationTSBufferMillis` equal to zero if that's the case.